server/utils/nunjucksSetup.ts

```diff
@@ -40,4 +45,28 @@ export default function nunjucksSetup(app: express.Express, applicationInfo: App
   )
 
   njkEnv.addFilter('initialiseName', initialiseName)
+
+  // convert errors to format for GOV.UK error summary component
+  njkEnv.addFilter('errorSummaryList', (errors = []) => {
```

Comment:
These two Nunjucks filters for preparing data for GDS error components (and accompanying unit tests) are largely copied from BaPV staff UI but updated to work with version 7 of `express-validator`.